### ISSUE TITLE
Uncaught TypeError: this.settingsView.destroy is not a function

### ISSUE THREAD

**Issue by RobbeR** [gwomacks/php-debug#116]
[Enter steps to reproduce below:]

1. I just clicked on the PHP Debug button

**Atom Version**: 1.4.3
**System**: Mac OS X 10.11.2
**Thrown From**: [php-debug](https://github.com/gwomacks/php-debug) package, v0.2.1

### Stack Trace

Uncaught TypeError: this.settingsView.destroy is not a function

```
At /Users/myUser/.atom/packages/php-debug/lib/php-debug.coffee:253

TypeError: this.settingsView.destroy is not a function
    at Object.module.exports.PhpDebug.toggleDebugging (/Users/myUser/.atom/packages/php-debug/lib/php-debug.coffee:253:21)
    at atom-workspace.subscriptions.add.atom.commands.add.php-debug:toggleDebugging (/Users/myUser/.atom/packages/php-debug/lib/php-debug.coffee:102:93)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1013:28)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
```

### Commands

```
     -7:44.8.0 tabs:close-all-tabs (li.tab.sortable.right-clicked)
  3x -6:10.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  2x -5:18 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -1:00.8.0 tabs:close-all-tabs (li.tab.sortable.right-clicked)
  2x -0:36.1.0 php-debug:toggleBreakpoint (atom-text-editor.editor)
     -0:02.4.0 php-debug:toggleDebugging (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "destroyEmptyPanes": false,
    "openEmptyEditorOnStart": false
  },
  "php-debug": {}
}
```

### Installed Packages

```coffee
# User
git-diff-details, v0.21.0
php-debug, v0.2.1

# Dev
No dev packages
```

**Comment by RobbeR** [comment]
This issue solved by install the xdebug-helper extension in Chrome

**Comment by cchamplin** [comment]
Hopefully fixed in v0.2.2 - Released Today
